File: Java Labs/Patterns On Java/src/com/pattern/flyweigh/SharedClothingItem.java
package com.pattern.flyweigh;

public class SharedClothingItem implements ClothingItem {
    private String type;

    public SharedClothingItem(String type) {
        this.type = type;
    }

    @Override
    public void display(String color) {
        System.out.println("Type: " + type + ", Color: " + color);
    }
}

File: Python Labs/com/pattern/flyweigh/SharedClothingItem.py
from com.pattern.flyweigh.ClothingItem import ClothingItem

class SharedClothingItem(ClothingItem):
    def __init__(self, item_type):
        self.item_type = item_type

    def display(self, color):
        print(f"Type: {self.item_type}, Color: {color}")